.github/workflows/actions_docs.yml

```diff
@@ -0,0 +1,31 @@
+name: sphinx
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  docs:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
```

Comment:
Latest version for `actions/checkout` is v3, and `actions/setup-python` is v4. Is there a specific reason to use the older actions?

Reply:
No reason, actions are now updated to latest versions.